### DISCUSSION TITLE
Swup stuck on transition

### DISCUSSION THREAD

**Discussion by CHolitas** [swup/swup#396] [Help] [answered]
So... I have this problem with Swup where happens two things:

A) At first swup-enabled doesn't appear on the HTML class and...
B) When I enter on another of the links, Swup appears to work, except for the fact that it gets stucked on is-animating

Can someone help me?

Quick Side note: This appears on the console command: ` Cannot set properties of null (setting 'responseURL') Swup.min.js` does this have something to do with this problem?

Another side note: I'm using localhost

**Comment by AugustMiller** [comment]
Do you have a `<title>` tag in the head of your response?

**Reply by CHolitas** [reply]
What? I don't understand, in my HTML file yes I have a `<title>`. But otherwise I solved it, i just needed to move the script to the end  of the body, but thanks for responding. ;)

**Comment by daun** [comment, marked as answer]
This was most likely a missing container. The most recent version 2.0.17 adds more helpful error messages that will make debugging this type of issue much easier.
